src/DeliveryCountry/DeliveryItaly.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Answear\InpostBundle\DeliveryCountry;
+
+class DeliveryItaly implements DeliveryCountryInterface
+{
+    private const BASE_URL = 'https://api-it-local-points.easypack24.net/';
```

Comment:
Consider saving the link in env and then you don't have to change the bundle every time a new market develops. Am I right that only the base url is different?

Reply:
Good idea, let's use configProvider